pr_best_practices.py

```diff
@@ -49,6 +49,16 @@ def check_pr_description_not_empty(description):
         sys.exit(1)
 
 
+def check_pr_description_contains_jira(description):
+    regex = r"JIRA: \[[A-Z]*-[0-9]*\]\(https:\/\/issues.redhat.com\/browse\/[A-Z]*-[0-9]*\)"
```

Comment:
```suggestion
    regex = r"JIRA: \[[A-Z]+-[0-9]+\]\(https:\/\/issues.redhat.com\/browse\/[A-Z]+-[0-9]+\)"
```

A bit of hardening. :)